includes/migration/wc/class-fs-wc-migration.php

```diff
@@ -981,24 +765,24 @@ protected function get_purchase_vat_for_api() {
 		protected function get_purchase_for_api() {
 			$purchase                         = array();
 			$purchase['billing_cycle']        = 0;
-			$purchase['payment_method']       = $this->get_local_purchase_gateway();
-			$purchase['customer_external_id'] = 'edd_customer_' . $this->_edd_customer->id;
-			$purchase['license_key']          = self::$_edd_sl->get_license_key( $this->_edd_license->ID ); // Preserve the same keys.
+			$purchase['payment_method']       = $this->_wc_order->payment_method;
```

Comment:
@shramee what are the optional values of `$this->_wc_order->payment_method`? Because FS only accepts `paypal` and `cc` (for credit cards).